pegasus/sites.v3/code.org/public/educate/science/videos.md

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
videos on this page are a little too big and hang off the right side 
<img width="361" alt="screen shot 2017-04-03 at 1 41 16 pm" src="https://cloud.githubusercontent.com/assets/12300669/24630907/45092694-1873-11e7-9fc3-9b6041c64f49.png">